### ISSUE TITLE
runtime.d.ts type interface EnvironmentInfo field buildType real value field is buildtype

### ISSUE THREAD

**Issue by daodao97** [wailsapp/wails#1539]
### Description

```ts
// wailsjs/runtime/runtime.d.ts
export interface EnvironmentInfo {
    buildType: string;
    platform: string;
    arch: string;
}
```

EnvironmentInfo field `buildType` real value field is `buildtype`

if use `env.buildType`, will get undefind

### To Reproduce

wails dev in template

### Expected behaviour

```ts
export interface EnvironmentInfo {
    buildtype: string;
    platform: string;
    arch: string;
}
```
or 
```ts
export interface EnvironmentInfo {
    buildType: string;
    platform: string;
    arch: string;
}

 EnvironmentInfo()  // { buildtype: "dev" }  => { buildType: "dev" }
```

### Screenshots

<img width="397" alt="image" src="https://user-images.githubusercontent.com/15009280/177757936-2ec3d050-a4fe-4989-b564-bd39957f8d80.png">

### System Details

```shell
Wails CLI v2.0.0-beta.38


Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:		MacOS
Version: 	11.6
ID:		199506
Go Version:	go1.18
Platform:	darwin
Architecture:	amd64

Wails
------
Version: 	v2.0.0-beta.38

Dependency			Package Name	Status		Version
----------			------------	------		-------
xcode command line tools 	N/A		Installed	2384
npm 				N/A		Installed	8.0.0
*upx 				N/A		Installed	upx 3.96
*nsis 				N/A		Available

* - Optional Dependency
```

**Comment by leaanthony** [comment]
Nice pickup. Fix on master.

**Comment by leaanthony** [comment]
@daodao97 - Any feedback?

**Comment by daodao97** [comment]
I'm sorry for not replying in time, it is fixed on `v2.0.0-beta.40`, thanks very much.
